src/rprblender/properties/render.py

```diff
@@ -479,7 +485,12 @@ def init_rpr_context(self, rpr_context, is_final_engine=True, use_gl_interop=Fal
                     # only enable metal once and if a GPU is turned on
                     metal_enabled = True
                     context_flags |= {pyrpr.CREATION_FLAGS_ENABLE_METAL}
-                        
+
+        if isinstance(rpr_context, pyhybrid.Context) and self.hybrid_low_mem:
```

Comment:
rpr_context can never be of type pyhybrid.Context. It could have type engine.context.RPRContext, engine.context.RPRContext2 or engine.context_hybrid.RPRContext.
Please fix checking with correct type.